alea/models/blueice_extended_model.py

```diff
@@ -21,27 +21,31 @@ class BlueiceExtendedModel(StatisticalModel):
     This class extends the `StatisticalModel` class and provides methods
     for generating data and computing likelihoods based on blueice.
 
+    Attributes:
+        parameters (Parameters): Parameters object containing the parameters of the model.
+        data (dict): Data of the statistical model.
+        is_data_set (bool): Whether data is set.
+        _likelihood (LogLikelihoodSum): A blueice LogLikelihoodSum instance.
+        likelihood_names (list): List of likelihood names.
+        data_generators (list): List of data generators for each likelihood term.
```

Comment:
Same as in the base clas.. I think I'd limit this to attributed we want people to use.

Reply:
We can trust the users to have the basic knowledge that the underscore indicates that this is a private attribute. The pros to keep them here is to remind developers of the existence of these attributes. What do you think?

Reply:
Yes you're right. Let's leave it like this for now, maybe we can add a reference to the methods that should be used in the docstrings like write that `_generate_data` is the special implementation and should be called as `generate_data` defined in the mother class.